### ISSUE TITLE
Hulu Error RUNUNK13

### ISSUE THREAD

**Issue by ianthrive** [slashlos/Helium#30]
With Helium 2, I got the "Unsupported Browser" error, so I'm trying this version. When trying to play this:
https://www.hulu.com/watch/c05513f5-d01e-41bd-9492-ddd4d540cae1

I get this:
> Hulu Error Code: RUNUNK13

I'm using releast version 3.0.5787.

I'll try to attach a screenshot.

**Comment by slashlos** [comment]
Yes, I've seen similar before, related to webkit api, setup or their sdk is needed.  You might try to start in Safari, then Share the webpage to Helium.
